api/src/core/nickname.generator.ts

```diff
@@ -0,0 +1,24 @@
+import { CrockfordBase32 } from 'crockford-base32';
+import seedrandom from 'seedrandom';
+
+import { adjectives } from '@root/resource/nickname/adjectives.json';
+import { animals } from '@root/resource/nickname/animals.json';
+import { colors } from '@root/resource/nickname/colors.json';
+import { jobs } from '@root/resource/nickname/jobs.json';
+
+export function generateUserNickname(userId: string): string {
+  const randomNumberGenerator = seedrandom(userId);
+  const randomNumber = randomNumberGenerator();
+  const adjectiveIdx = Math.floor(randomNumber * adjectives.length);
+  const jobIdx = Math.floor(randomNumber * jobs.length);
+  const colorIdx = Math.floor(randomNumber * colors.length);
+  const animalIdx = Math.floor(randomNumber * animals.length);
```

Comment:
random number 는 한 번 만들어서 네 곳에서 재활용하는 방식이 아닌, 네 번 생성하는 방식으로 바꿔야 할 것 같아요.

```suggestion
  const rand = seedrandom(userId);
  const adjectiveIdx = Math.floor(rand() * adjectives.length);
  const jobIdx = Math.floor(rand() * jobs.length);
  const colorIdx = Math.floor(rand() * colors.length);
  const animalIdx = Math.floor(rand() * animals.length);
```

Reply:
확인했습니다. https://github.com/MovieReviewComment/Mr.C/pull/99/commits/cd6f366d5e7710d418333701564b0134e9a8f782 에서 반영했습니다.